### PR TITLE
fix: Rev @crxjs/vite-plugin to properly include side_panel(s)

### PR DESCRIPTION
**Pull request by idearat**
The current version of this template predates the addition of side_panel support to the @crxjs/vite-plugin. Adjusting the version to the latest beta (18) fixes this and allows builds to properly include side_panel content.

### CODE REVIEW

**Comment by idearat** [comment]
Actually I realized that just updating package*.json would only be half the solution. So I've updated the PR with the sample side_panel implementation I used to verify it all works. Hope this is helpful!

File: src/background/index.ts [hunk 1]
```diff
@@ -3,3 +3,8 @@ import { storage } from "../storage";
 chrome.runtime.onInstalled.addListener(() => {
     storage.get().then(console.log);
 });
+
+// Allows users to open the side panel by clicking on the action toolbar icon
+chrome.sidePanel.setPanelBehavior({ openPanelOnActionClick: true })
```
Comment (NekitCorp) [inline src/background/index.ts:8]: Unfortunately this code conflicts with `src/popup/popup.html` which also opens by clicking on the action toolbar icon :(

image

Comment (idearat) [reply src/background/index.ts]: Updated to offer the side panel variation as an option, but commented out.

Comment (NekitCorp) [reply src/background/index.ts]: Great! Thanks you!

**Pull request merged**